Jenkinsfile

```diff
@@ -54,6 +54,15 @@ webappPipeline {
         readJSON(file: 'dist/manifest.json')
     }
 
+    deployConfig = [
+      dev : 'always',
+      test : 'always',
+      prod : 'always',
+      'fedramp-use2-core': 'always'
+    ]
+
+    autoSubmitCm = true
```

Comment:
I didn't see a problem with this. I think it makes sense if we have a `master` build to auto submit the CM. Especially since we would have already released the npm version.